lib/premailex/html_parser/meeseeks.ex

```diff
@@ -6,7 +6,8 @@ defmodule Premailex.HTMLParser.Meeseeks do
   require Logger
   import Meeseeks.CSS
   alias Premailex.HTMLParser
-  alias Meeseeks.{Selector.CSS.Parser.ParseError.Document}
+  alias Meeseeks.{Selector.CSS.Parser.ParseError}
+  alias Meeseeks.Document
```

Comment:
If you want, you can leave `Document` in the previous line, just separate it from the other module with a comma. I.e.
```elixir
alias Meeseeks.{Selector.CSS.Parser.ParseError, Document}
```
although I think it's more obvious the way you changed it.

Reply:
Yeah, it's fine with the two aliases. You can remove the curly brackets, I forgot to remove them as I was working on the Meeseeks module.